rest-api/category/models/category-models.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../../../database/dbConfig')
+
+module.exports = {
+    findAllCategories,
+    findCategoryBy,
+    addCategory
+}
+
+async function findAllCategories() {
+    const categories = await db('categories');
+    return categories;
+}
```

Comment:
This is nice because it's following the async await however by declaring a variable and then returning it, you're cutting a bit out inefficiency, also it doesn't have to be async until you need to use promises `.then `

Reply:
Doesn't `db('categories')` return a promise?

Reply:
Yes I stand corrected